From 90cf8e084b81a0a66a5765f8f20f237797a19336 Mon Sep 17 00:00:00 2001
From: Mallory Daries <mallory.daries@openwide.fr>
Date: Tue, 10 Mar 2015 14:10:36 +0100
Subject: [PATCH 2/2] decrease pcie_set_read_rq size from 4096 to 256

---
 drivers/ata/sata_sil24.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/ata/sata_sil24.c b/drivers/ata/sata_sil24.c
index 59f0d63..e728b8e 100644
--- a/drivers/ata/sata_sil24.c
+++ b/drivers/ata/sata_sil24.c
@@ -1336,7 +1336,7 @@ static int sil24_init_one(struct pci_dev *pdev, const struct pci_device_id *ent)
 	/* Set max read request size to 4096.  This slightly increases
 	 * write throughput for pci-e variants.
 	 */
-	pcie_set_readrq(pdev, 4096);
+	pcie_set_readrq(pdev, 1024);
 
 	sil24_init_controller(host);
 
-- 
1.9.1

